README.rst

```diff
@@ -58,153 +58,30 @@ Using with Content Libraries on a Tutor Devstack
 The easiest way to try out the "Content Libraries v2" feature along with Blockstore is to use the Tutor devstack and
 `this library-authoring MFE Tutor plugin <https://github.com/openedx/frontend-app-library-authoring/pull/50>`_. See that plugin's README for details.
 
---------------------------
-Using with Docker Devstack
---------------------------
-
-Prerequisite: Have your Open edX `Devstack <https://github.com/openedx/devstack>`_ properly installed and working. Your devstack must be tracking the ``master`` branch of ``edx-platform``; using Blockstore on an older devstack release is not supported.
-
-#. Clone this repo and ``cd`` into it.
-
-#. To start the django development server inside a docker container, run this on
-   your host machine:
-
-   .. code::
-
-      make easyserver
-
-   Blockstore is now running at http://localhost:18250/ . Now we need to configure Studio/LMS to work with it.
-
```

Comment:
Can  you move these lines down under `Running Tests`? We should keep that section, and this info on how to run the blockstore docker container, so that people still know how to run tests.

Reply:
Is it important to still have that way to run tests? I was under the impression that all the tests are currently run in "app mode" in the edx-platform test suite, and these are only needed to run the tests in "IDA mode".

Actually, can we soon remove "IDA mode" from edx-platform and blockstore entirely, or is there some reason we need to continue to support both options?

Reply:
I've but those back in, in a modified form. Please review :)

Reply:
Thanks for retaining those test instructions until we can get the standalone service removed.

edx.org is still running blockstore standalone in production, so we shouldn't remove it yet.